spec/controllers/idv/how_to_verify_controller_spec.rb

```diff
@@ -94,6 +106,14 @@
         irs_reproofing: false,
       }.merge(ab_test_args)
     end
+
+    before do
+      allow_any_instance_of(Idv::Session).to receive(:service_provider).
+        and_return(ServiceProvider.last)
+      allow_any_instance_of(ServiceProvider).to receive(:in_person_proofing_enabled).
+        and_return(true)
+    end
```

Comment:
Arguably, we don't need to add new "when the ServiceProvider has IPP disabled" contexts to these later tests, because we're testing `confirm_step_allowed` above proving that the user never gets here.

But we do need the SP to have the flag true because otherwise we can't get here in the first place.

This and the following two blocks are directly copied-and-pasted and... aren't ideal... but we'll make it nice after we make it work.